lib/tooling_invoker/invoke_docker.rb

```diff
@@ -35,8 +35,8 @@ def call
         exit_status: json['exit_status']
       }
 
-      job.result = json['result']
-      job.status = job.result ? 200 : 400
+      job.output = json['output_files']
```

Comment:
@ErikSchierboom I would like this to also use the `output_filepaths` else we could end up with a situation where a file gets passed through via this route, but not via the runc route. I'll merge but could you add that here, and add a comment explaining, in a separate PR pls?

Reply:
Will do